docs/docs/styled-components.md

```diff
@@ -112,7 +112,31 @@ export default () => (
 )
 ```
 
-### Enabling user stylesheets with a stable class name
+## Creating Global Styles
+
+Normally, styled components are created for a single CSS class that is isolated from other components. In some cases, it’s helpful (and cleaner!) to create a global style so more than one component can have access to the same styles. In a way, you can create a base stylesheet.
+
+A global style can be created by using `createGlobalStyle` and can be added to files like `container.js` or `index.js`.
```

Comment:
I've kinda said this in my rewrite of the sentences. Unsure if we need to callout `container.js` or `index.js` specifically.

```suggestion
```

Reply:
👍